packages/manager/modules/nutanix/src/dashboard/general-info/routing.js

```diff
@@ -34,11 +34,11 @@ export default /* @ngInject */ ($stateProvider) => {
         return promise;
       },
       packType: /* @ngInject */ (clusterTechnicalDetails) =>
-        clusterTechnicalDetails.license.edition,
+        clusterTechnicalDetails?.license?.edition,
       isLegacyPack: /* @ngInject */ (packType) =>
         LEGACY_PACK_TYPES.includes(packType),
       clusterAddOns: /* @ngInject */ (NutanixService, serviceInfo) =>
-        NutanixService.getClusterOptions(serviceInfo.serviceId),
+        NutanixService.getClusterOptions(serviceInfo.serviceId).catch(() => []),
```

Comment:
should be done only if `response.status === 403`, otherwise the error should be displayed